drools-ansible-rulebook-integration-protoextractor/src/main/java/org/drools/ansible/rulebook/integration/protoextractor/prototype/ValueExtractionVisitor.java

```diff
@@ -0,0 +1,80 @@
+package org.drools.ansible.rulebook.integration.protoextractor.prototype;
+
+import java.util.List;
+import java.util.Map;
+
+import org.drools.ansible.rulebook.integration.protoextractor.ast.ASTNode;
+import org.drools.ansible.rulebook.integration.protoextractor.ast.DefaultedVisitor;
+import org.drools.ansible.rulebook.integration.protoextractor.ast.ExtractorNode;
+import org.drools.ansible.rulebook.integration.protoextractor.ast.IdentifierNode;
+import org.drools.ansible.rulebook.integration.protoextractor.ast.IndexAccessorNode;
+import org.drools.ansible.rulebook.integration.protoextractor.ast.SquaredAccessorNode;
+import org.drools.model.Prototype;
+
+public class ValueExtractionVisitor extends DefaultedVisitor<Object> {
+    private final Object original;
+    private Object cur;
+
+    public ValueExtractionVisitor(Object original) {
+        this.original = original;
+        this.cur = this.original;
+    }
+
+    @Override
+    public Object defaultVisit(ASTNode n) {
```

Comment:
This method is never supposed to be invoked, so it should throw an `UnsupportedOperationException`.

Reply:
done with https://github.com/kiegroup/drools-ansible-rulebook-integration/pull/56/commits/0f41f50052adc3dc3a6668ab1375c6a805fff0e2